src/main/kotlin/wu/seal/jsontokotlin/model/ConfigManager.kt

```diff
@@ -90,4 +91,28 @@ object ConfigManager : IConfigManager {
             }
         }
 
+    var fileHeader: String
+        get() = if (TestConfig.isTestModel) {
+            TestConfig.fileHeadText
+        } else {
+            val fileTemplateManager = FileTemplateManager.getDefaultInstance()
+            val fileTemplate = fileTemplateManager.getDefaultTemplate(FileTemplateManager.FILE_HEADER_TEMPLATE_NAME)
+            fileTemplate?.let {
+                fileTemplate.getText(fileTemplateManager.defaultProperties)
+            }
+        }
+
+        set(value) {
```

Comment:
@hanshengjian  We may not need to set this value, right?

Reply:
yes